### PR TITLE
configs/imxrt1050-evk/loadable_elf_apps: enable SDRAM_PRIMARY and SPI

### PR DESCRIPTION
**Pull request by sunghan-chang**
update it using latest

Signed-off-by: sunghan-chang <sh924.chang@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [6b04e2ae196065f8492fb7b292ea955de7cb3b3c] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [6b04e2ae196065f8492fb7b292ea955de7cb3b3c] - Code Rule Check (C++) OK.

File: build/configs/imxrt1050-evk/loadable_elf_apps/defconfig [hunk 1]
```diff
@@ -457,7 +458,12 @@ CONFIG_DEV_NULL=y
 # CONFIG_PWM is not set
 CONFIG_ARCH_HAVE_I2CRESET=y
 # CONFIG_I2C is not set
-# CONFIG_SPI is not set
+CONFIG_SPI=y
+CONFIG_SPI_USERIO=y
```
Comment (btheosam) [inline build/configs/imxrt1050-evk/loadable_elf_apps/defconfig]: Is IO on spi needed?

Comment (sunghan-chang) [reply build/configs/imxrt1050-evk/loadable_elf_apps/defconfig]: Right. It is required by DA.

**Pull request merged**